config/crd/bases/infrastructure.cluster.x-k8s.io_ocimachinepools.yaml

```diff
@@ -936,7 +936,6 @@ spec:
                         type: string
                     type: object
                   isPvEncryptionInTransitEnabled:
-                    default: true
```

Comment:
Just thinking out loud: What happens if kubebuilder sets this to false as default so the CRD still defines a default? Not saying it has to be done, but having a default in the CRD is nicer for the user. 🤷‍♂️

Reply:
I think or bool, it is generally understood that false is default, our provider and other providers also uses this, so I think this is ok.